src/OpenDB/test/gcd_abstract_lef.lefok

```diff
@@ -803,8 +806,14 @@ MACRO gcd
   OBS
     LAYER metal1 ;
     RECT  0 0 200.26 201.6 ;
+    LAYER metal2 ;
+    RECT  0 0 200.26 201.6 ;
+    LAYER metal3 ;
+    RECT  0 0 200.26 201.6 ;
     LAYER metal4 ;
     RECT  0 0 200.26 201.6 ;
+    LAYER metal6 ;
```

Comment:
What happened to metal5, in the DEF the clock is routed on metal6 and below. I don't see anything obviously missing in the `findSWireLayerObstructions`  or `findWireLayerObstructions` that could ignore the clock.

Reply:
It looks like everything on metal5 in the def is part of a via? I'm not really the best at reading the def files. I skip anything that's a via, but maybe I shouldn't.

@maliberty Any advice on this front?

Reply:
In the power grid we may just via down across a layer without straps/stripes.  I think you'll need to consider the via shapes in this case.

Reply:
@arlpetergadfort I just pushed something that takes into account what Matt mentioned. Does that look closer to what you were looking for?

Reply:
Yes, this is good.